doctr/datasets/cord.py

```diff
@@ -65,14 +66,15 @@ def __init__(
                             x = word["quad"]["x1"], word["quad"]["x2"], word["quad"]["x3"], word["quad"]["x4"]
                             y = word["quad"]["y1"], word["quad"]["y2"], word["quad"]["y3"], word["quad"]["y4"]
                             if use_polygons:
+                                # x1, y1, x2, y2, x3, y3, x4, y4
```

Comment:
I would argue we have to specify this more: (x1, y1) points to which corner, and in what direction is the next one

Reply:
agree :+1: